ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -1372,6 +1549,77 @@ private Offset getLastRecordOffset(LogSegment segment) {
     return lastOffset;
   }
 
+  // hardDeletePauseAndResumeOnRestart() helpers
+
+  /**
+   * Utility to test hard delete pause and resume with restart.
+   * @param deleteShutdownFile {@code true} if shutdown file needs to be deleted during reload of index, {@code false}
```

Comment:
could you elaborate how deleting or not deleting the shutdown file affects this test?